tpl/bootstrap.php

```diff
@@ -41,7 +41,7 @@
 endif;
 ?>
 		<noscript><link type="text/css" rel="stylesheet" href="css/noscript.css" /></noscript>
-		<script type="text/javascript" data-cfasync="false" src="js/jquery-3.6.0.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+		<script type="text/javascript" data-cfasync="false" src="js/jquery-3.6.0.js" integrity="sha512-894YE6QWD5I59HgZOGReFYm4dnWc1Qt5NtvYSaNcOP+u1T9qYdvdihz0PPSiiqn/+/3e7Jo4EaG7TubfWGUrMQ==" crossorigin="anonymous"></script>
```

Comment:
Regarding your comment regarding jQuery. The old one _was_ the correct/official one. It should have worked. See https://releases.jquery.com/ and click on "minified" in for "jQuery Core 3.6.0". As that upgrade was contributed by someone else in #880 I explicitly commented and checked/compared it.
It is SHA256 of course, not SHA-512, but SHA-512 provides no real security advantage, but also no disadvantage, so it is of course okay to use.

Did you check that the jQuery file is the correct one manually?

Reply:
Okay, cloned it and checked the files themselve by downloading from https://code.jquery.com/jquery-3.6.0.min.js.

```
$ diff -s jquery-3.6.0.min.js jquery-3.6.0-privatebin.js
Files jquery-3.6.0.min.js and jquery-3.6.0-privatebin.js are identical
```

Reply:
Yes I did, that was what I tried to explain with the phrase "I did download the library from https://code.jquery.com/jquery-3.6.0.min.js and it is byte by byte the same as what already got committed"